### ISSUE TITLE
Google Oauth2 - PlatformException(CANCELED, User canceled login, null, null)

### ISSUE THREAD

**Issue by scadot** [teranetsrl/oauth2_client#50]
Hi,

I'm using your library to retrieve a google oauth2 token with Authorization Code flow.
The process is ok on IOS but each time I call the API on Android, i get the following error:
 **PlatformException(CANCELED, User canceled login, null, null)**

On the GCP, I create a OAuth 2.0 Client IDs for Android and get the following package name:
![package_name](https://user-images.githubusercontent.com/10340004/100252981-d7ced180-2f40-11eb-9d70-4f7599bd1233.png)

In my AndroidManifest.xml, I create a new Intent:
![intent](https://user-images.githubusercontent.com/10340004/100253191-15cbf580-2f41-11eb-98cb-ed4ce00f514e.png)

And I create the Google client API as below:
![googleoauth](https://user-images.githubusercontent.com/10340004/100253232-254b3e80-2f41-11eb-89c4-f9b9a7a84153.png)

I'm not sure to understand what I'm doing wrong.
Could you please help me to solve this problem?

Thanks.

**Comment by okrad** [comment]
Hi @scadot, can you try adding an additional "/" in the redirectUri (com.example.remote://oauth2redirect)?

**Comment by supermar1010** [comment]
I had the same problem for me it was fixed, by adding the activity tag for `com.linusu.flutter_web_auth.CallbackActivity`. And then uninstalling `flutter clean` and then reinstalling the app :)

**Comment by alexandranb** [comment]
Hi, if I add com.linusu.flutter_web_auth.CallbackActivity to my app, the app crashes after install. Did anyone have this issue with it?

**Comment by supermar1010** [comment]
Without a stacktrace it is hard to say what went wrong. Did you completely reinstall the app after adding the lib and the Manifest entry?

**Comment by alexandranb** [comment]
Yes, added com.linusu.flutter_web_auth.CallbackActivity to manifest. Cleaned app, built it, uninstalled it, and ran it. After installment it briefly displayed the splashscreen and then it automatically closed. Nothing appears in debug.
